src/Microsoft.OpenApi/Models/OpenApiCallback.cs

```diff
@@ -35,6 +35,22 @@ public class OpenApiCallback : IOpenApiSerializable, IOpenApiReferenceable, IOpe
         /// </summary>
         public IDictionary<string, IOpenApiExtension> Extensions { get; set; } = new Dictionary<string, IOpenApiExtension>();
 
+        /// <summary>
+        /// Parameter-less constructor
+        /// </summary>
+        public OpenApiCallback() { }
+
+        /// <summary>
+        /// Initializes a copy of an <see cref="OpenApiCallback"/> object
+        /// </summary>
+        public OpenApiCallback(OpenApiCallback callback)
+        {
+            PathItems = callback.PathItems;
```

Comment:
```suggestion
            PathItems = new(callback.PathItems);
```
(and all the other ones, expect for strings)

Thanks for reverting to copy constructor, with the current implementation, you're copying references. That means if a ObjectB is a copy of ObjectA and ObjectA.Foo = "bar", ObjectB.Foo also = "bar", which could have a lot of side effects.